src/utils/daos_hdlr.c

```diff
@@ -1640,46 +1655,33 @@ cont_query_hdlr(struct cmd_args_s *ap)
 			DP_OID(ap->oid));
 	}
 
-	if (ap->path != NULL) {
-		/* cont_op_hdlr() already did resolve_by_path()
-		 * all resulting fields should be populated
-		 */
-		assert(ap->type != DAOS_PROP_CO_LAYOUT_UNKOWN);
-
-		printf("DAOS Unified Namespace Attributes on path %s:\n",
-			ap->path);
```

Comment:
Just curious why we don't want to print this anymore?

Reply:
the type, chunk size and oclass are not really properties of the UNS, but the container itself.. i wasn't sure why we had it in the first place.